#onshape origin: x=325.375000, y=158.048750, z=19 inches
grid_locations:
  # {type, y}
  # rotation = 180 for red, 0 for blue
  # closest to origin
  pos1: ["cone", 0.519] # cone
  pos2: ["cube", 1.078] # cube, tag id 6 blue or 3 red
  pos3: ["cone", 1.637] # cone
  # coop grid
  pos4: ["cone", 2.195] # cone
  pos5: ["cube", 2.754] # cube, tag id 7 blue or 2 red
  pos6: ["cone", 3.313] # cone
  # third grid
  pos7: ["cone", 3.872] # cone
  pos8: ["cube", 4.431] # cube, tag id 8 blue or 1 red
  pos9: ["cone", 4.989] # cone

map_to_tag:
  # oops this was already made identically in 2023_apriltag_locations...
  # x,y,z , dont care about rotation I think
  tag1: [15.513558, 1.071626, 0.462788]
  tag2: [15.513558, 2.748026, 0.462788]
  tag3: [15.513558, 4.424426, 0.462788]
  tag4: [16.178784, 6.749796, 0.695452]
  tag5: [0.36195, 6.749796, 0.695452]
  tag6: [1.02743, 4.424426, 0.462788]
  tag7: [1.02743, 2.748026, 0.462788]
  tag8: [1.02743, 1.071626, 0.462788]

# x distance from the apriltag on the cube node to the end of the things that stick out between nodes
# aka the distance to be back from the tag
# if this is too low we will hit the wall
offset_from_cube_tag_to_end: 0.5

hold_position_timeout: 0.5 # seconds